powersimdata/design/investment/tests/test_investment_costs.py

```diff
@@ -97,18 +97,20 @@ def test_calculate_ac_inv_costs(mock_grid):
 
 
 def test_calculate_dc_inv_costs(mock_grid):
-    expected_dc_cost = 10 * 679.1799258421203 * 457.1428571 * calculate_inflation(
-        2015
-    ) + 135e3 * 10 * 2 * calculate_inflation(  # lines
-        2020
-    )  # terminals
+    expected_dc_cost = (
+        # lines
+        10 * 679.1799258421203 * 457.1428571 * calculate_inflation(2015)
+        # terminals
+        + 135e3 * 10 * 2 * calculate_inflation(2020)
+    )
     dc_cost = _calculate_dc_inv_costs(mock_grid)
     assert dc_cost == pytest.approx(expected_dc_cost)
```

Comment:
There are no changes here. Just reorganizing the lines, correct?

Reply:
Correct.